### PR TITLE
Create 'enableTimeFeatureType' function

### PR DESCRIPTION
**Pull request by chrismayer**
Adds the function `enableTimeFeatureType` in the layers class in order to enable time dimension for a vector based layer (`FeatureType`).

Please review @jakobMiksch

### CODE REVIEW

File: src/layer.js [hunk 1]
```diff
@@ -340,4 +340,68 @@ export default class LayerClient {
       return false;
     }
   }
+
+  /**
+   * Enables TIME dimension for the given FeatureType layer.
+   *
+   * @param {String} workspace Workspace where layer to enable time dimension for is in
```
Comment (JakobMiksch) [inline src/layer.js:347]:
```suggestion
   * @param {String} workspace Workspace containing layer to enable time dimension for
```

File: src/layer.js [hunk 2]
```diff
@@ -340,4 +340,68 @@ export default class LayerClient {
...
+   * @param {String} datastore The datastore where the FeatureType to enable time dimension for is in
```
Comment (JakobMiksch) [inline src/layer.js:348]:
```suggestion
   * @param {String} datastore The datastore containing the FeatureType to enable time dimension for```

File: src/layer.js [hunk 3]
```diff
@@ -340,4 +340,68 @@ export default class LayerClient {
...
+   * @param {String} name FeatureType to enable time dimension for
+   * @param {String} attribute Data column / attribute holding the time values
+   * @param {String} presentation Presentation type: 'LIST' or 'DISCRETE_INTERVAL' or 'CONTINUOUS_INTERVAL'
+   * @param {Number} resolution Resolution in milliseconds, e.g. 3600000 for 1 hour
+   * @param {String} defaultValue The default time value, e.g. 'MINIMUM' or 'MAXIMUM' or 'NEAREST' or 'FIXED'
+   * @param {Boolean} [nearestMatchEnabled] Enable nearest match
+   * @param {Boolean} [rawNearestMatchEnabled] Enable raw nearest match
+   * @param {String} [acceptableInterval] Tolerance interval for nearest mach (e.g. 'PT30M'), only has an effect if 'nearestMatchEnabled' is true
```
Comment (JakobMiksch) [inline src/layer.js:356]:
```suggestion
   * @param {String} [acceptableInterval] Tolerance interval for nearest mach (e.g. 'PT30M'), only has an effect if 'nearestMatchEnabled' is true
   
   @returns {Boolean} If TIME dimension could be enabled
```

**Review by JakobMiksch — APPROVED** [review]
Looks good. I added comments regarding the docs. Otherwise ready to merge.

**Comment by chrismayer** [comment]
Thanks for the review @JakobMiksch. I addressed all your comments in 18aef3bc18d. Will merge now.

**Pull request merged**